schema/1.5/example.json

```
@@ -0,0 +1,101 @@
+{
+  "id": "SERIAL-2018-08-01-112",
+  "$template": "BLOCKCHAIN-ACADEMY-2018-001",
+  "description": "This masters is awarded to developers who can blockchain",
+  "issuedOn": "2018-08-01T00:00:00+08:00",
+  "expiresOn": "2118-08-01T00:00:00+08:00",
+  "name": "Master of Blockchain",
+  "admissionDate": "2017-08-01T00:00:00+08:00",
+  "graduationDate": "2018-08-01T00:00:00+08:00",
+  "issuers": [
+    {
+      "name": "Blockchain Academy",
+      "did": "DID:SG-UEN:U18274928E",
+      "url": "https://blockchainacademy.com",
+      "email": "registrar@blockchainacademy.com",
+      "certificateStore": "0xd9580260be45c3c0c2fb259a82f219b513054012",
+      "additionalProp": "0x0000000000000000000000000000000000000000000000000000000000000000"
+    }, 
+    {
+      "name": "School of Crypto-economics",
+      "url": "https://ceschool.sg",
+      "certificateStore": "0xA33ddAEE02369D18A21E29dD2E7A12d65eE671d2"
+    }
+  ],
+  "recipient": {
+    "name": "Mr Blockchain",
+    "did": "DID:SG-NRIC:S99999999A",
+    "email": "mr-blockchain@gmail.com",
+    "phone": "+65 88888888",
+    "additionalProp": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "nric": "S0000001Z",
+    "studentId": "1232"
+  },
+  "qualificationLevel": [{
+    "frameworkName": "singapore/ssec-eqa",
+    "frameworkVersion": "2015-level-2",
+    "levelCode": "51",
```

```suggestion
    "code": "51",
```

It's already in the `qualificationLevel` object